libsolidity/codegen/ir/IRGeneratorForStatements.cpp

```diff
@@ -1722,6 +1749,13 @@ std::ostream& IRGeneratorForStatements::define(IRVariable const& _var)
 	return m_code;
 }
 
+std::ostream& IRGeneratorForStatements::assign(IRVariable const& _var)
```

Comment:
Can't this be reused by the function above?

Reply:
`m_code << "let " << assign(_var)` - wouldn't that look weird?

Oh and I also think it's not possible because of the way `<<` works.

Reply:
Hm ok